packages/app/scripts/build.js

```diff
@@ -140,7 +145,7 @@ function build(previousSizeMap) {
 
     if (stats.hasWarnings()) {
       console.warn(chalk.yellow('Build warnings:'));
-      stats.compilation.warnings.forEach(({ name, message }) => {
+      getStatsWarnings(stats).forEach(({ name, message }) => {
```

Comment:
**What's going on here?**

Now that there are multiple entrypoints, we actually get back a [`MultiStats`](https://webpack.js.org/api/node/#stats-object) object which has the same API, except `compilation` is nested under a `stats` array.